### ISSUE TITLE
I was told to report a bug

### ISSUE THREAD

**Issue by bumberboy** [vikejs/vike#1232]
### Description

```
  [vike@0.4.143][request(3)][Bug] You stumbled upon a bug in Vike's source code. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error; a maintainer will fix the bug (usually under 24 hours).
at file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/serializePageContextClientSide.js:49:17
  at Array.forEach (<anonymous>)
at serializePageContextClientSide (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/serializePageContextClientSide.js:40:22)
at getPageContextTag (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets/getHtmlTags.js:127:48)
at htmlTag (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets/getHtmlTags.js:113:28)
at resolveHtmlTag (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets/injectHtmlTags.js:54:42)
at file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets/injectHtmlTags.js:13:25
  at Array.map (<anonymous>)
at file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets/injectHtmlTags.js:13:14
  at Array.forEach (<anonymous>)
at injectHtmlTags (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets/injectHtmlTags.js:10:15)
at injectToHtmlBegin (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets.js:41:17)
at injectHtmlTagsToString (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/injectAssets.js:11:18)
at renderDocumentHtml (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/html/renderHtml.js:38:32)
at executeOnRenderHtmlHook (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/renderPage/executeOnRenderHtmlHook.js:33:24)
at renderPageAlreadyRouted (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/renderPage/renderPageAlreadyRouted.js:56:30)
at renderPageNominal (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/renderPage.js:257:36)
at renderPageAlreadyPrepared (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/renderPage.js:113:45)
at renderPageAndPrepare (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/renderPage.js:93:12)
at file:///projectdirectory/node_modules/vike/dist/esm/node/plugin/shared/getHttpRequestAsyncStore.js:68:35
  at renderPage (file:///projectdirectory/node_modules/vike/dist/esm/node/runtime/renderPage.js:46:50)
at file:///projectdirectory/server/index.ts:53:29

```

**Comment by bumberboy** [comment]
was following this guide: https://connectrpc.com/docs/web/ssr/#server-load-functions

but did not want to use `toPlainMessage`. So the object being passed had methods that were probably not serializable.

**Comment by brillout** [comment]
Fix pre-released in `0.4.144-commit-f7ab002`.

@bumberboy Would your company be up for [sponsoring](https://github.com/sponsors/brillout)?
